### ISSUE TITLE
Search auth fails on private tabs

### ISSUE THREAD

**Issue by srirambv** [brave/brave-browser#16517]
## Description 
Search auth fails on private tabs

## Steps to reproduce 

   1. Set Google as default
   2. Authenticate Brave search on normal tab via email auth link
   3. Open search.brave.com and click/dismiss the default search
   4. Go to settings and set Brave as default for normal tab, private tab let it be Google
   5. Open a new private tab and visit search.brave.com
   6. Throws auth error on private tab
   7. Go back to normal tab and perform search on brave works fine

## Actual result 
Search auth fails on private tabs

## Expected result
Once auth is done should work irrespective of normal/private tabs

## Issue reproduces how often 
Easy

## Version/Channel Information:

- Can you reproduce this issue with the current Play Store version? NA
- Can you reproduce this issue with the current Play Store Beta version? Not sure
- Can you reproduce this issue with the current Play Store Nightly version? Yes

## Device details
   - Install type (ARM, x86): ARM
   - Device type (Phone, Tablet, Phablet): Phone
   - Android version: 10
   
## Brave version
1.28.11

### Website problems only
- Does the issue resolve itself when disabling Brave Shields? NA
- Does the issue resolve itself when disabling Brave Rewards?NA
- Is the issue reproducible on the latest version of Chrome? NA

### Additional information

cc: @bsclifton

**Comment by bsclifton** [comment]
This should work great now (after today's go-live)- can you confirm @srirambv?

**Comment by bsclifton** [comment]
Actually, will close as there isn't auth anymore - so the problem mentioned isn't applicable anymore. We can create a new issue in the future if there's an auth specific problem

**Comment by srirambv** [comment]
Confirmed this works now without the auth process
